src/api/apiClients/axios.ts

```diff
@@ -0,0 +1,59 @@
+import type { IApiClient, IRequestOptions, IResponse } from "../../types/api/apiClient.types.js";
+import axios, { AxiosRequestConfig, AxiosResponse } from "axios";
+import FormData from "form-data";
+import { logApiStep } from "../../utils/reporter/decorators.js";
+
+const request = axios.create();
+let response: AxiosResponse;
+
+class AxiosApiClient implements IApiClient {
+
+  @logApiStep
+  async sendRequest<T>(options: IRequestOptions): Promise<IResponse<T>> {
+    options.timeout ? options.timeout : 120000;
+    try {
+      response = await request(options as AxiosRequestConfig);
```

Comment:
I think it is possible to return this.transformResponse from this line

Reply:
Removed this realization